etna/whatson/models.py

```diff
@@ -867,6 +879,13 @@ class ExhibitionPage(
         max_num=1,
     )
 
+    video_title = models.CharField(
```

Comment:
If there is no default here (which I don't think there should be) then this should maybe be a required field or at least default to something (content design help needed)

Reply:
I've made this non-required so editors can leave it blank if they want to, but if they've added a video it becomes required (flags up when saving the page)